docker/compose.dev.yml

```diff
@@ -36,7 +34,7 @@ services:
     image: traefik:v2.10
     container_name: traefik-gateway
     ports:
-      - "9000:80"
+      - "80:80"
     volumes:
       - "../servers/config/gateway/traefik.yml:/etc/traefik/traefik.yml"
```

Comment:
The next three lines correspond to traefik using file-based config. Please remove them. It needs to be replaced with pure docker-style config.